source/cn/wiki/design-principles.md

```diff
@@ -7,3 +7,15 @@ version: 2.1
 ---
 
 <!-- toc -->
+
+# Weex设计理念
+这篇文章想从Weex的设计理念入手，让你对Weex的设计哲学、优势有一个全面的了解，希望可以帮助Weex的使用者更好的在适合的场景应用Weex，也希望Weex的开发者可以在设计理念的指引下将Weex变得更好
```

Comment:
The content is great. As for the formatting, I think it's better to add a space between English words and Chinese characters.